packages/react-mutation-mapper/src/model/MutationMapperStore.ts

```diff
@@ -1,12 +1,17 @@
 // TODO do not use generated models
```

Comment:
let's remove this `TODO` or change it to `TODO use generated genome nexus models as well`. Actually it might be better to create a separate ticket for that.